### PR TITLE
UCS/SYS/TOPO: Added bw estimation for Sapphire Rapids family - v1.15.x

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Added estimation of bandwidth for Intel Sapphire Rapids family.
## Why ?
To avoid TCP lane selection instead of IB lanes.
https://github.com/openucx/ucx/issues/9287

### CODE REVIEW

**Review by brminich — APPROVED** [review]

File: src/ucs/sys/topo/base/topo.c [hunk 1]
```diff
@@ -308,6 +308,9 @@ static void ucs_topo_sys_root_distance(ucs_sys_dev_distance_t *distance)
 {
     distance->latency = 500e-9;
     switch (ucs_arch_get_cpu_model()) {
+    case UCS_CPU_MODEL_INTEL_SAPPHIRERAPIDS:
+        distance->bandwidth = 23288 * UCS_MBYTE;
```
Comment (ivankochin) [inline src/ucs/sys/topo/base/topo.c:312]: How did you get this number?

Comment (rakhmets) [reply src/ucs/sys/topo/base/topo.c:312]: The value is copied from the issue. This is a value of bandwidth for message size 262144.

**Review by ivankochin — DISMISSED** [review]

**Review by yosefe — DISMISSED** [review]

**Comment by ivankochin** [comment]
Minor: It seems like we need to update this table too https://github.com/openucx/ucx/blob/master/src/tools/info/sys_info.c#L25

**Comment by rakhmets** [comment]
> Minor: It seems like we need to update this table too https://github.com/openucx/ucx/blob/master/src/tools/info/sys_info.c#L25

Done. Also squashed commits because of the changes are minor.

**Review by yosefe — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Pull request merged**